docs/src/modules/components/ApiPage/sections/PropertiesSection.js

```diff
@@ -1,155 +0,0 @@
-/* eslint-disable react/no-danger */
-import * as React from 'react';
-import PropTypes from 'prop-types';
-import Box from '@mui/material/Box';
-import { useTranslate } from '@mui/docs/i18n';
-import { SectionTitle } from '@mui/docs/SectionTitle';
-import ToggleDisplayOption, {
-  useApiPageOption,
-} from 'docs/src/modules/components/ApiPage/sections/ToggleDisplayOption';
-import PropertiesList, { getHash } from 'docs/src/modules/components/ApiPage/list/PropertiesList';
-import PropertiesTable from 'docs/src/modules/components/ApiPage/table/PropertiesTable';
-
-export function getPropsToC({
-  componentName,
-  componentProps,
-  inheritance,
-  themeDefaultProps,
-  t,
-  hash,
-}) {
-  return {
-    text: t('api-docs.props'),
-    hash: hash ?? 'props',
-    children: [
-      ...Object.entries(componentProps)
-        .filter(([, propData]) => propData.description !== '@ignore')
-        .map(([propName]) => ({
-          text: propName,
-          hash: getHash({ propName, componentName }),
-          children: [],
-        })),
-      ...(inheritance
-        ? [{ text: t('api-docs.inheritance'), hash: 'inheritance', children: [] }]
-        : []),
-      ...(themeDefaultProps
-        ? [{ text: t('api-docs.themeDefaultProps'), hash: 'theme-default-props', children: [] }]
-        : []),
-    ],
-  };
-}
-
-export default function PropertiesSection(props) {
-  const {
-    properties,
-    propertiesDescriptions,
-    componentName = '',
-    showOptionalAbbr = false,
-    title = 'api-docs.props',
-    titleHash = 'props',
-    level = 'h2',
-    spreadHint,
-    hooksParameters = false,
-    hooksReturnValue = false,
-    defaultLayout,
-    layoutStorageKey,
-  } = props;
-  const t = useTranslate();
-
-  const [displayOption, setDisplayOption] = useApiPageOption(layoutStorageKey, defaultLayout);
-  const formatedProperties = Object.entries(properties)
-    .filter(([, propData]) => propData.description !== '@ignore')
```

Comment:
If you take a close look, you will notice this line is removed in the `tsx` files. That's because the `description` property does not exist in the typing.

I will need to double-check with all components using this. But for example, base uses the `@ignore` on some props, and they never reach that place.